src/FsAutoComplete/LspServers/AdaptiveServerState.fs

```diff
@@ -418,7 +433,7 @@ type AdaptiveState(lspClient: FSharpLspClient, sourceTextFactory: ISourceTextFac
 
 
   let runAnalyzers (config: FSharpConfig) (parseAndCheck: ParseAndCheckResults) (volatileFile: VolatileFile) =
-    async {
+    asyncEx {
```

Comment:
should the direct use of `async` be a code smell for us now?

Reply:
Probably, though we can make additional PRs to hit all those places.